gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -73,6 +73,10 @@
 
 /**
  * Google Cloud Storage {@link FileSystemProvider} implementation.
+ *
+ * <p><b>Note:</b> This class should not be used directly ever. This class is instantiated by the
```

Comment:
Nit: could rephrase the first sentence to say "This class should never be used directly"